Rubberduck.RibbonDispatcher/Sample Ribbons/SampleRibbon/src/MyResourceManager.cls

```diff
@@ -0,0 +1,109 @@
+VERSION 1.0 CLASS
+BEGIN
+  MultiUse = -1  'True
+END
+Attribute VB_Name = "MyResourceManager"
+Attribute VB_GlobalNameSpace = False
+Attribute VB_Creatable = False
+Attribute VB_PredeclaredId = False
+Attribute VB_Exposed = False
+Option Explicit
+
+Private Const ModuleName As String = "MyResourceManager."
+
+Private Type TRibbonStrings
+    Strings As Rubberduck_RibbonDispatcher.ControlStrings
```

Comment:
Wait a minute, so those are... labels? Why aren't they called *labels* then?

Reply:
Because for each Control there is a specific string called a **Label**; another specific string called a **ScreenTip**; a third specific string called a **SuperTip**; a fourth specific string called a **KeyTip**; and potentially fifth and sixth specific strings called **Description** and **Tag**. To not use the Ribbon terminology would be - uber confusing for everyone involved.

Reply:
Ok, then it's not the name (`ControlStrings` seems fine considered that), it's the way the API exposes it as a *collection of strings*, or some kind of dictionary, when a friendly API would expose dedicated properties for each one, e.g. `Label`, `ScreenTip`, `SuperTip`, `KeyTip` and then `Description` and `Tag`, with the implementations being no-op / returning an empty string when not applicable. No?